doc/can-query-logic.md

```diff
@@ -14,139 +14,141 @@ data caching and real-time behavior.
 
 @signature `new QueryLogic( [schemaOrType] [,options] )`
 
-The `can-query-logic` package exports a constructor function that builds _query logic_
-from:
+  The `can-query-logic` package exports a constructor function that builds _query logic_
+  from:
 
-- an optional schema or type argument, and
-- an optional `options` argument used to convert alternate parameters to
+  - an optional schema or type argument, and
+  - an optional `options` argument used to convert alternate parameters to
   the expected [can-query-logic/query] format.
 
 
-For example, the following builds _query logic_ from a [can-define/map/map]:
+  For example, the following builds _query logic_ from a [can-define/map/map]:
 
-```js
-import {DefineMap, QueryLogic} from "can";
+  @sourceref ./can-query-logic-models/todo-example.js
+  @codepen
+  @highlight 3-10,only
 
-const Todo = DefineMap.extend({
-    id: {
-        identity: true,
-        type: "number"
-    },
-    name: "string",
-    complete: "boolean"
-});
+  Once a _query logic_ instance is created, it can be used to
+  perform actions using [can-query-logic/query queries].  For example,
+  the following might select 20 incomplete todos from a list of todos:
 
-var todoQueryLogic = new QueryLogic(Todo);
-```
+  @sourceref ./can-query-logic-models/todo-example.js
+  @codepen
+  @highlight 14-24,only
 
-Once a _query logic_ instance is created, it can be used to
-perform actions using [can-query-logic/query queries].  For example,
-the following might select 20 incomplete todos from a list of todos:
-
-```js
-// Perform query logic:
-todoQueryLogic.filterMembers({
-    filter: {
-        complete: false
-    },
-    sort: "-name",
-    page: {start: 0, end: 19}
-},[
-    {id: 1, name: "do dishes", complete: false},
-    {id: 2, name: "mow lawn", complete: true},
-    ...
-]) //-> [matching records]
-```
+  By default `can-query-logic` supports queries represented by the [can-query-logic/query]
+  format.  It supports a variety of operators and options.  It looks like:
 
-By default `can-query-logic` supports queries represented by the [can-query-logic/query]
-format.  It supports a variety of operators and options.  It looks like:
+  ```js
+  import {QueryLogic} from "can";
+  import {Todo} from "//unpkg.com/can-demo-models@5";
 
-```js
-{
```

Comment:
I think it's better to show what the query format looks like instead of have the more complex example.